bin/git-effort

```diff
@@ -191,6 +187,17 @@ if test "${#paths}" -eq 0; then
   unset save_ifs
 fi
 
+max=0
+for path in "${paths[@]}"; do
+    cur=${#path}
+    if [[ $max -lt $cur ]]; then
+        max=$cur
+    fi
+done
+# set column width to match longest filename
```

Comment:
IMO this comment makes more sense above `max=0`, since that's the logic for finding the longest filename.

Reply:
@nfischer 
Tweaked in the new commit.